app/src/org/odk/collect/android/widgets/SelectOneAutoAdvanceWidget.java

```diff
@@ -90,7 +90,7 @@ public SelectOneAutoAdvanceWidget(Context context, FormEntryPrompt prompt) {
 
                 RadioButton r = new RadioButton(getContext());
                 r.setOnCheckedChangeListener(this);
-                r.setText(prompt.getSelectChoiceText(mItems.get(i)));
+                r.setText(stylize(prompt.getSelectChoiceText(mItems.get(i))));
```

Comment:
right - this'll just return the same string but cast to a spannable string at the moment I believe?